cob_hardware_config/cob3-2/config/teleop.yaml

```diff
@@ -1,49 +1,62 @@
+#cob3-2
+##########
 # common params
 run_factor: 2.0
+joy_num_buttons: 12
+joy_num_axes: 6
+joy_num_modes: 4
+mode_switch_button: 8
+light_topic_name: /light
+
+# axes
+axis_vx: 1
+axis_vy: 0
+axis_vz: 5
+axis_roll: 4
+axis_pitch: 3
+axis_yaw: 2
 
 # buttons
-lower_neck_button: 6
-upper_neck_button: 4
-tray_button: 3
-arm_joint12_button: 0
-arm_joint34_button: 1
-arm_joint56_button: 2
-arm_joint7_button: 3
-base_safety_button: 4
 deadman_button: 5
+safety_button: 4
+init_button: 9
```

Comment:
why? the number is the parameter, not the parameter name and it depends on the joystick, if we sort it by number we will have a different parameters order for each robot. 
And for me it is better define always first the deadman_button , without this parameter you can't move any component, and then the other buttons that are not required.